### PR TITLE
feat(glossary): relational-database

### PR DESCRIPTION
**Pull request by amyshwang**
- Adding relational data base glossary entry
- Linking to this page from blog posts

### CODE REVIEW

File: src/pages/glossary/relational-database.md [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+  same balance even after a power outage.
+
+## Relational databases vs. non-relational databases
+
+The table below compares the traits of relational and non-relational databases:
+
+|                  | Relational Databases              | Non-relational Databases                                    |
+| ---------------- | --------------------------------- | ----------------------------------------------------------- |
+| Data Model       | Relational                        | Key-value, document, graph, search, time-series             |
+| Query Language   | SQL                               | Varies                                                      |
```
Comment (javier) [inline src/pages/glossary/relational-database.md:101]: Would it make sense to mention query language can be SQL on some non-relational databases?

File: src/pages/glossary/relational-database.md [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+| Use Cases        | Structured data, complex querying | Unstructured or semi-structured data, high write-throughput |
+| Data Consistency | Strong consistency (ACID)         | Eventual consistency                                        |
+| Examples         | MySQL, PostgreSQL, SQL Server     | MongoDB, Cassandra, Redis, DynamoDB                         |
```
Comment (javier) [inline src/pages/glossary/relational-database.md:104]: Do we want to add QuestDB here?

File: src/pages/glossary/relational-database.md [hunk 3]
```diff
@@ -0,0 +1,118 @@
...
+
+## Popular relational databases
+
+- [MySQL](https://www.mysql.com)
+- [PostgreSQL](https://www.postgresql.org)
+- [SQLite](https://www.sqlite.org/index.html)
+- [Microsoft SQL Server](https://www.microsoft.com/en-GB/sql-server/sql-server-downloads)
+- [Oracle](https://www.oracle.com/)
+- [MariaDB](https://mariadb.org/)
+- [Amazon Aurora](https://aws.amazon.com/rds/aurora/)
```
Comment (javier) [inline src/pages/glossary/relational-database.md:114]: Do we want to add QuestDB here?

**Review by javier — APPROVED** [review]

File: src/pages/glossary/relational-database.md (hunk 3 above)
Comment (amyshwang) [reply src/pages/glossary/relational-database.md:114]: I'd leave it as it is as we are primarily time-series database and we have articles about migrating from a relational database to tsdb (although we have relational model...).
Here I will list only the classical relational database if it makes sense...

**Review by javier — APPROVED** [review]

**Pull request merged**